python/lsst/cp/pipe/measureCrosstalk.py

```diff
@@ -474,7 +479,7 @@ def run(self, inputRatios, inputFluxes=None, camera=None, inputDims=None, output
         combinedFluxes = defaultdict(lambda: defaultdict(list))
         for ratioDict, fluxDict in zip(inputRatios, inputFluxes):
             for targetChip in ratioDict:
-                if calibChip and targetChip != calibChip:
+                if calibChip and targetChip != calibChip and targetChip != calibDetector.getName():
                     raise RuntimeError("Received multiple target chips!")
```

Comment:
Can that error message give some feedback as to which of the three checks failed?

Reply:
Updated to be clearer.